app/src/androidTest/java/HomePage.kt

```diff
@@ -1,8 +1,6 @@
 import com.bijoysingh.quicknote.R
 
-open class HomePage : EspressoBaseTest() {
+open class HomePage {
```

Comment:
It shouldn't be open, as I mentioned previously.
Read this: https://blog.mindorks.com/understanding-open-keyword-in-kotlin